src/configs/accessories/use-32-microsteps

```diff
@@ -0,0 +1,20 @@
+opt_set DEFAULT_AXIS_STEPS_PER_UNIT "{ 200, 200, 800, 200 }"
+
+# NB! The values below will only have any effect if the board is fitted with programmable drivers
```

Comment:
Are you interested in using this for any TMC configurations? This adds a lot of changes.

Reply:
I guess it doesn't matter.

Reply:
If you're referring to the number of changes to Configuration_adv.h then opt_set will change this for all TMC sections at once. Not sure if it's worth attempting to be more targeted here.